_sass/_components/_logo.scss

```diff
@@ -3,4 +3,6 @@
   height: 150px;
   max-width: 150px;
   float: right;
+
+  box-shadow: container(box-shadow, 2);
```

Comment:
This seems like an extraneous commit to this pull request, but there's no harm in including it here either